src/styles/getMuiTheme.js

```diff
@@ -224,4 +226,12 @@ export default function getMuiTheme(baseTheme, muiTheme) {
       borderColor: palette.borderColor,
     },
   }, muiTheme);
+
+  if (computedTheme.isRtl) {
```

Comment:
Right right :grin: I'm really bad with left and right, before and after :D 

> I think we should consider removing the dependencies on isRtl from the components.

I think this will be very limiting. Some logics related to rtl is simply not implementable with style modification. I wouldn't change this behavior :grin: